content/boilerplates/notes/1.2.md

```diff
@@ -0,0 +1,64 @@
+## Installation
+
+- **Added** pod anti-affinity and toleration support to all of our control plane components.
+- **Added** `sidecarInjectorWebhook.neverInjectSelector` and `sidecarInjectorWebhook.alwaysInjectSelector` to allow users to further refine workloads that have sidecar automatically injected or not, based on label selectors.
+- **Added** `global.logging.level` and `global.proxy.logLevel` to allow users to easily configure logs for control plane and data plane components globally.
+- **Added** support to configure the Datadog location via `global.tracer.datadog.address`.
+
+## Upgrades
+
+- **Removed** deprecated Adapter and Template CRDs by default. Configure `mixer.templates.useTemplateCRDs=true` and `mixer.adapters.useAdapterCRDs=true` to re-enable them.
+
+Refer to the [installation option change page](/docs/reference/config/installation-options-changes/) to view the complete list of changes.
+
+## General
+
+- **Added** `traffic.sidecar.istio.io/includeInboundPorts` annotation to eliminate the need for service owner to declare `containerPort` in the deployment yaml file.
```

Comment:
Need to document whether this will become the default in the next release. Want to encourage users to do this.

Reply:
Done